packages/block-editor/src/components/block-list/content.scss

```diff
@@ -453,7 +453,8 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 	}
 }
 
-.block-editor-iframe__body {
+.block-editor-iframe__html {
 	transition: all 0.3s;
+	background-color: $gray-300;
```

Comment:
This background color appears to affect the entire editor canvas. See #59348

Reply:
Yes, we'll have to only apply it for zoom out